bundle/manifests/mtk-operator.clusterserviceversion.yaml

```diff
@@ -62,6 +74,19 @@ spec:
           - get
           - patch
           - update
+        - apiGroups:
```

Comment:
Can we add a comment on why we need access to this?

Reply:
Since CSV is autogenerated all the comments are getting omitted, put a comment explaining the need of the permission along with kubebuilder markers for rbac [here](https://github.com/konveyor/mtk-operator/pull/3/files#diff-2b629fd9df1b62c27ddabca5d7b0e2986e68889bba4c5bf1472481fc3afc0f54R37).